kroxylicious-schema-validation/src/main/java/io/kroxylicious/proxy/filter/schema/SchemaValidationFilterContributor.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy.filter.schema;
+
+import io.kroxylicious.proxy.filter.FilterContributor;
+import io.kroxylicious.proxy.filter.KrpcFilter;
+import io.kroxylicious.proxy.filter.schema.validation.ProduceRequestValidator;
+import io.kroxylicious.proxy.service.BaseContributor;
+
+import static io.kroxylicious.proxy.filter.schema.JsonValueValidator.isValidJson;
+
+public class SchemaValidationFilterContributor extends BaseContributor<KrpcFilter> implements FilterContributor {
+
+    public static final BaseContributorBuilder<KrpcFilter> FILTERS = BaseContributor.<KrpcFilter> builder()
+            .add("ValidJson", () -> new ProduceValidationFilter(new ProduceRequestValidator(isValidJson())));
```

Comment:
Similarly the configuration model would be different. Maybe this should be `WellFormedJson`, for example.